core/src/schema/content/string.rs

```diff
@@ -565,8 +565,18 @@ impl Compile for StringContent {
             }) => {
                 let begin = begin
                     .clone()
-                    .unwrap_or_else(|| ChronoValue::default_of(ChronoValue::origin(), *type_));
-                let end = end.clone().unwrap_or(begin.clone() + Duration::weeks(52));
+                    .unwrap_or_else(|| ChronoValue::default_of(ChronoValue::now(), *type_));
+                let end = end
+                    .clone()
+                    .unwrap_or_else(|| ChronoValue::default_of(ChronoValue::now(), *type_));
+                if begin > end {
```

Comment:
This check duplicates [L475](https://github.com/getsynth/synth/blob/4cce7809105c6579740cf3588c1f049a89c7b011/core/src/schema/content/string.rs#L475).